### PR TITLE
GUUI 3334 modify review status to show pending uwg review

### PR DESCRIPTION
**Pull request by KrupaRami**
# Description

An approach for modifying request status when it is in uwg_review step. Please let me know your thoughts or if there are any changes needed.

This is a PR to support the GES DISC workflow, specifically to modify a "Request Status" when workflow is in UWG review step. It should show "Pending UWG Review" instead of Pending Review when workflow is in uwg review step.

The change supports adding step_status_label to the step table and passing it with value to the uwg review step in GES DISC workflow.

## Spec

See Ticket: [https://bugs.earthdata.nasa.gov/browse/GUUI-3334](https://bugs.earthdata.nasa.gov/browse/GUUI-3334)

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.

## Change Log

_(copy/paste-able change log notes. check the box when the change is also in CHANGELOG.md)_

* [Add _____](link to commitid)
* [Fix _____](link to commitid)

### CODE REVIEW

**Comment by camposeddie** [comment]
@KrupaRami @DavisGauntUAH Have the requested changes of this PR been resolved? Is this PR ready to be validated for merging?

**Comment by KrupaRami** [comment]
> @KrupaRami @DavisGauntUAH Have the requested changes of this PR been resolved? Is this PR ready to be validated for merging?
I have pushed the requested changes.

**Comment by camposeddie** [comment]
Ready for testing once https://github.com/eosdis-nasa/earthdata-pub-dashboard/pull/50 is ready

**Comment by camposeddie** [comment]
@KrupaRami @joncarlson Is the step label working for you locally? In validating, I'm still just seeing the default label with these branch changes.

**Comment by KrupaRami** [comment]
Yes it is working for me locally. step label only appears when step is in uwg review step.

File: src/nodejs/lambda-handlers/workflow.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ async function createStep(step, stepName, rollbackInfo) {
   return db.workflow.createStep({
     step_name: stepName,
     type: step.type,
+    step_status_label: step.step_status_label,
```
Comment (camposeddie) [inline src/nodejs/lambda-handlers/workflow.js:14]: I think this was added after Davis' comment:
```
Fields in the step table need to be able to be edited using the workflow editor. Please reach out if you need help adding support for this.
```
however, this doesn't address the comment. For example, https://github.com/eosdis-nasa/earthdata-pub-api/blob/2d858252591f616cffe749993a6f040dfa83950b/src/nodejs/lambda-layers/database-util/src/query/workflow.js#L41 was not updated, so effectively this addition is never actually used.

Comment (KrupaRami) [reply src/nodejs/lambda-handlers/workflow.js:14]: ohh I see, I will update the createStep function and push the changes soon.

**Review by camposeddie — APPROVED** [review]

**Pull request merged**